CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed to support Helm 3; To keep using Helm 2, please use the version under 0.2.2.
```

Comment:
Is this should be added or changed? Since this is break changes, I prefer to have it as changed.

Reply:
```suggestion
- Updated to support Helm 3; To keep using Helm 2, please use version 0.2.X.
```

Good point. What about this?  If we have to make fixes for Helm 2 the version will be 0.2.X.

Reply:
Thanks, that makes sense.